### PR TITLE
Unit test "polygonal-face-tessellation"

### LINKED ISSUE #61: Example file "polygonal-face-tessellation" not rendered.
The file is [here](../blob/development/testdata/IFC4x1ExampleFiles/official/E4_Tessellated_shape/polygonal-face-tessellation.ifc).  OIP doesn't show anything.

### LINKED ISSUE #317: [BUG] Official IfcPolygonalFaceSet example is wrong
**Describe the bug**
See [here](https://forums.buildingsmart.org/t/ifcpolygonalfaceset-and-face-winding/3169/3)

**Expected behavior**
- [ ] Correct the example file, so that face winding is correct.
- [ ] Update unit tests if needed.

### PR DESCRIPTION
**Pull request by Elvira2227**
Added new unit test "polygonal-face-tessellation"

Fixes #61 closes #317

P.S.: Please check Line#18 with describe in #338

### CODE REVIEW

File: UnitTests/Schemas/IFC4X1/polygonal-face-tessellation/src/polygonal-face-tessellation.cpp [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+	
+	buw::storeImage(testPath("polygonal-face-tessellation_front_left_bottom.png").string(), image_front_left_bottom);
+	buw::storeImage(testPath("polygonal-face-tessellation_front_right_bottom.png").string(), image_front_right_bottom);
+	buw::storeImage(testPath("polygonal-face-tessellation_top_left_front.png").string(), image_top_left_front);
+	buw::storeImage(testPath("polygonal-face-tessellation_top_front_right.png").string(), image_top_front_right);
+	buw::storeImage(testPath("polygonal-face-tessellation_top_left_back.png").string(), image_top_left_back);
+	buw::storeImage(testPath("polygonal-face-tessellation_top_right_back.png").string(), image_top_right_back);
+	buw::storeImage(testPath("polygonal-face-tessellation_back_left_bottom.png").string(), image_back_left_bottom);
+	buw::storeImage(testPath("polygonal-face-tessellation_right_bottom_back.png").string(), image_right_bottom_back);
+
```
Comment (pjanck) [inline UnitTests/Schemas/IFC4X1/polygonal-face-tessellation/src/polygonal-face-tessellation.cpp]: In the end version, these lines should be commented out.

**Review by pjanck — CHANGES_REQUESTED** [review]
Line 18 looks fine. Please include the screen shots as well.

File: UnitTests/Schemas/IFC4X1/polygonal-face-tessellation/src/polygonal-face-tessellation.cpp (hunk 1 above)
Comment (Elvira2227) [reply UnitTests/Schemas/IFC4X1/polygonal-face-tessellation/src/polygonal-face-tessellation.cpp]: Resolved 1681521

**Comment by SamuilsRulovs** [comment]
Something is wrong with screenshots, this example file should look like this: image

**Review by pjanck — CHANGES_REQUESTED** [review]
There is apparently something wrong in the geometry conversion.

**Comment by pjanck** [comment]
Notes:

- Implement faces with voids:

https://github.com/tumcms/Open-Infra-Platform/blob/90e74889a85fb7d1044419a0297c657c95d38355/Core/src/IfcGeometryConverter/FaceConverter.h#L1277-L1278

- Make use of this algorithm (perhaps make it a special function that is used by the original function and yourself):

https://github.com/tumcms/Open-Infra-Platform/blob/90e74889a85fb7d1044419a0297c657c95d38355/Core/src/IfcGeometryConverter/GeomUtils.cpp#L221-L260

**Comment by pjanck** [comment]
Superseded by #448 and #459

**Pull request closed without merging**